web/public/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8">
     <link rel="shortcut icon" href="https://tilt.build/favicon.ico">
+    <link href="https://fonts.googleapis.com/css?family=Inconsolata:400,700|Montserrat:600" rel="stylesheet">
```

Comment:
I've read that using Google fonts is really bad for performance. Is that true?

Reply:
i wouldn't be too worried about it right now